src/component/Calculator.js

```diff
@@ -0,0 +1,44 @@
+import CalculatorDigit from "./CalculatorDigit";
+import CalculatorOperation from "./CalculatorOperation";
+import { DIGITS, OPERATIONS } from "../constant";
+import useCalculate from "../hook/useCalculate";
+
+const Calculator = () => {
+  const { total, enterDigit, enterOperator, calculateTotal, clear } =
+    useCalculate();
+
+  return (
+    <div className="calculator">
+      <h1 id="total">{total}</h1>
+
+      <div className="digits flex">
+        {DIGITS.map((digit) => (
+          <CalculatorDigit
+            digit={digit}
+            key={digit}
+            onClick={() => enterDigit(digit)}
+          />
+        ))}
+      </div>
+
+      <div className="modifiers subgrid">
+        <button className="modifier" onClick={clear}>
+          AC
+        </button>
+      </div>
+
+      <div className="operations subgrid">
+        {OPERATIONS.map((operation, index) => (
+          <CalculatorOperation
+            operation={operation}
+            key={index}
```

Comment:
```suggestion
            key={operation}
```

operation에 중복이 없을테니 key로 써도 괜찮을거 같은데요..?

Reply:
key에는 고유한 숫자만 될거라고 착각하고 있었어요.. 고유한 문자열이면 사용 가능한거였군요!

Reply:
네~ key는 데이터의 고유한 값을 사용하는것이 가장 권장되는 패턴입니다~!